viewmaster/pdsiterator.py

```
@@ -245,13 +397,32 @@ def __next__(self):
+
+        Returns:
+            tuple: A tuple (logical_path, display_path, level=1) where:
+                - logical_path (str): Full logical path of the file.
+                - display_path (str): Parent directory path plus basename.
+                - level (int): Always 1 (cousin level).
+
+        Raises:
+            StopIteration: If no adjacent parent directory is available.
+        """
 
         prev_logical_path = self.current_logical_path
```

Ruff (0.14.10)

Local variable `prev_logical_path` is assigned to but never used

Remove assignment to unused variable `prev_logical_path`

(F841)